processor/Cargo.toml

```diff
@@ -26,8 +26,10 @@ std = ["vm-core/std", "winter-prover/std"]
 tracing = { version = "0.1", default-features = false, features = [
     "attributes",
 ] }
-vm-core = { package = "miden-core", path = "../core", version = "0.9", default-features = false }
 miden-air = { package = "miden-air", path = "../air", version = "0.9", default-features = false }
+static_assertions = "1.1.0"
+thiserror = { version = "1.0", default-features = false }
```

Comment:
Yeah we should use the fork anywhere in this workspace that we depend on `thiserror`, at least temporarily - the `error_in_core` feature is stabilized in 1.80, and I expect that the mainline `thiserror` will support no-std shortly thereafter. With that in mind, it probably is not worth moving the `thiserror` fork to our org for such a short time. That said, I'll double check and see what the plan is though for `thiserror`, and if it looks like it'll be a month or two, then moving it makes more sense (I don't actually care whether we move it now or not, but didn't want to overcomplicate things for what amounts to a temporary workaround).

We probably _should_ move the `miette` fork to our org though, since that will likely stick around for awhile longer, even once the `thiserror` fork goes away. I can try to find some time to do that.

Reply:
> the `error_in_core` feature is stabilized in 1.80

Ah - I didn't realize it is already slated for 1.80. In this case, no need to create `miden-thiserror` - we can just rely on your fork until the canonical `thiserror` is updated.